### DISCUSSION TITLE
Corne has no functional middle thumb buttons / How do I change files to accommodate? [QMK]

### DISCUSSION THREAD

**Discussion by Korkaner** [manna-harbour/miryoku#295] [Q&A] [answered]
Hello everyone,
I am quite new with QMK and Split KBs.

It seems like during soldering I broke the middle thumb button on both sides of the Corne PCB. As I can't really order new parts and redo it, I am trying my best to find accommodation.

I was really interested in using Miryoku with a Colemak-DH Layout, but I am a bit unsure with my options and how to edit the files to find a proper solution.

If this is not the proper channel to ask for help, then I apologize in advance for my rudeness.

**Comment by manna-harbour** [comment, marked as answer]
If you were missing top or bottom row inner index or pinkie column keys, or tertiary thumb keys, you can use key emulation combos. See https://github.com/manna-harbour/miryoku/issues/56. But the home thumb keys are fundamental and can't be substituted.

In this situation you should try to repair the damage, possibly a bad solder joint (reflow), broken trace (jump), or broken switch or diode (replace).

**Reply by Korkaner** [reply]
Thank you for the answer!

I already tried to reflow, use a different switch and replace the diode. So I may have a broken trace and in the long term, I will need to find a solution.
